Field.py

```diff
@@ -103,11 +103,17 @@ def __init__(self, screen, hex_width=25, field_size=None, hex_amount=None):
 
         # draw the water sprites on the screen
         self.water_group.draw(screen.get_screen())
+        # save the water sprite as one image for faster loading
+        pygame.image.save(screen.get_screen(), "Data/Sprites/water.bmp")
+        self.water_img = pygame.image.load("Data/Sprites/water.bmp").convert()
```

Comment:
maybe use surfaces instead, seems to work for me

self.water_surface = pygame.Surface(self.screen.get_size())
self.water_group.draw(self.water_surface)
self.screen.get_screen().blit(self.water_surface, (0,0))

Reply:
dit werkt bij mij niet: ik krijg een blank/wit water surface